### DISCUSSION TITLE
Feature Idea: Import Google Docs into Obsidian Vaults

### DISCUSSION THREAD

**Discussion by madpin** [coddingtonbear/obsidian-web#199] [Ideas]
Hey there,

I was thinking it would be awesome if the “obsidian-web” plugin could let us bring in documents from Google Docs straight into our vaults.

It could work like this:

	1.	PDF Import: Let us import Google Docs as PDFs, so we keep the original look and layout.
	2.	Full Document Import: Even better, let us import Google Docs as full, editable documents. This way, we can keep working on them in Obsidian.

Having this feature would make life a lot easier for those of us who use both Google Docs and Obsidian. It would help us keep everything in one place and streamline our workflow.

Thanks for considering this idea. I think it would be a great addition to the plugin and really help out a lot of users.

**Comment by coddingtonbear** [comment]
This doesn't sound like it overlaps enough with the use cases this extension is intended to solve.  I think it might be up to you to write your own extension implementing that kind of feature!
